tests/integration/cairo1_compiler_bindings/library_functions_test/prepare_test.cairo

```diff
@@ -40,7 +54,12 @@ fn test_prepare_tp() {
 fn test_prepare_no_args() {
    let mut arr = ArrayTrait::new();
    match prepare_tp(123, arr) {
```

Comment:
We should use prepare instead of prepare_tp here. Will address it in separate PR and create issue.